pkg/wireguard/agent/agent.go

```diff
@@ -48,6 +49,17 @@ import (
 
 const (
 	listenPort = 51871
+
+	// wireguardOverhead is an approximation for the overhead of wireguard
+	// encapsulation.
+	//
+	// https://github.com/torvalds/linux/blob/master/drivers/net/wireguard/device.c#L262:
```

Comment:
Nit: maybe you could pin the URL to a specific tag, e.g. 5.12.

Reply:
Done.